### ISSUE TITLE
eth_getLogs filters other than fromBlock/toBlock are not working at all

### ISSUE THREAD

**Issue by rcastellaj** [ledgerwatch/erigon#5547]
#### System information

Erigon version: erigon version 2022.09.03

OS & Version: Linux Alpine 3.15.6

Commit hash : 

v2022.09.03 tag

#### Expected behaviour

Running eth_getLogs with a topic and/or address filter effectively returns log entries that match the said filter. The range of blocks has events that match the filter criteria, and actually return correctly on a geth node.

#### Actual behaviour

Running eth_getLogs over curl with either the use of 'topics' or 'address' for filtering results in an empty array result. Any combination of address or topics is resulting in an empty array, single topic or multi topic filtering the same.

#### Steps to reproduce the behaviour

```
curl --location --request POST --header 'Content-Type: application/json' --data-raw '{"jsonrpc":"2.0","method":"eth_getLogs","params":[{"fromBlock":"0x4C4B40","toBlock":"0x4C4B41","topics":["0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef"]}],"id":0}' bsc-node-erigon:8545
```

The same above query works with Geth. Any combination of topics as ["hash"] or []["hash"], or even the use of "just the address filter" fail completely.

**Comment by hexoscott** [comment]
Hi @rcastellaj, I recently put a PR up to help with an issue where multiple subscriptions that contained any address/topic combo would effectively stop sending anything at all to either subscription.  I'm not sure timing wise if this made it into devel/alpha before you reported the issue.  If possible could you try your subscriptions on a build from Devel or alpha v2022.09.02 (or greater) and let me know if this is still an issue?  Thanks!

**Comment by revitteth** [comment]
This issue was fixed in v2022.09.02+, will close and can be reopened if we're still seeing the problem.
